layer/nrlf/core/tests/test_validators.py

```diff
@@ -485,6 +502,7 @@ def test_validate_category_coding_display_mismatch_observations():
     }
 
 
+# TODO: FIX name of this test to multiple codings
```

Comment:
any reason this hasn't been done already?

Reply:
Making the code review easier 🤣  I can change it once you had a complete view of the changes. Otherwise the diff is going to look ugly